packages/cozy-sharing/locales/en.json

```diff
@@ -62,6 +62,13 @@
       "sync_to_mine": "Synchronise in my Cozy",
       "add_to_mine": "Add to my Cozy",
       "close": "Close"
+    },
+    "permissionLink": {
+      "title": "Allow anyone with the link to :",
```

Comment:
no space in english ;)

https://www.learnenglish.de/punctuation/punctuationtext.html#colon